packages/studio-ui/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -141,6 +142,8 @@ class Diagram extends Component<Props> {
       zoomToLevel: this.props.zoomToLevel
     })
 
+    setInterval(() => this.props.refreshFlowsLinks(), 1000)
```

Comment:
I know this needs to change before merging but I didn't find the solution to refresh the diagram. Maybe someone knows how to do it properly.

Reply:
@allardy? Personally, I'm not sure how it can be done and honestly, the redux code of the flow editor is kind of a mess but we could probably "listen" to changes in the flow `currentHashes` and then update the links or pass some kind of props to your components so that it can trigger a function that handles correctly updating the links upon update.

Reply:
Why don't you pass the refreshFlowLinks to the toolbar so you can call it once youre done?

Reply:
I quickly tried it and calling `refreshFlowsLinks` directly after the flow update does not work. Maybe it's because the update is done async and there is no way, in the layout components, to know that it is done updating!